### PR TITLE
MYFACES-4570: fix NPE

### PR DESCRIPTION
**Pull request by pnicolucci**
https://issues.apache.org/jira/browse/MYFACES-4570

### CODE REVIEW

**Comment by tandraschko** [comment]
+1
we could do a Objects.equals instead?

**Comment by pnicolucci** [comment]
> +1 we could do a Objects.equals instead?

sure, let me push up a change.

**Comment by pnicolucci** [comment]
fyi looks like our integration tests are broken (unrelated to this PR):

`Error:    IntegrationTest.testPlainHtml » Runtime Unable to instantiate Drone via org.openqa.selenium.chrome.ChromeDriver(ChromeOptions): org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: This version of ChromeDriver only supports Chrome version 108`

**Comment by pnicolucci** [comment]
@tandraschko  look better? Thanks for the review!

File: impl/src/main/java/org/apache/myfaces/resource/FacesJSResourceLoader.java [hunk 1]
```diff
@@ -95,7 +96,7 @@ public InputStream getResourceInputStream(ResourceMeta resourceMeta)
                 getHttpServletRequest(FacesContext.getCurrentInstance().getExternalContext());
 
         //name mapping does not happen on meta level
-        if(!(libraryName.equals(JsfLibrary.NAMESPACE) &&
+        if(!Objects.equals(libraryName,null) && !(libraryName.equals(JsfLibrary.NAMESPACE) &&
```
Comment (melloware) [inline impl/src/main/java/org/apache/myfaces/resource/FacesJSResourceLoader.java:99]:
```suggestion
        if(!JsfLibrary.NAMESPACE.equals(libraryName)) &&
```

Joshua Bloch always recommends flipping to use the Constant first this way you can't have an NPE

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by pnicolucci** [comment]
@melloware should have been obvious to me :) Thanks! Look good now?

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
Yep and I sent @bohmber a note its saying something about Drone only supporting Chrome 108 and I bet GitHub Actions Linux is is on Chrome 110.  I don't fully understand the Drone stuff.

**Pull request merged**